parallel-install/pkg/metadata/metadata.go

```diff
@@ -110,17 +135,17 @@ func (p *Provider) WriteKymaDeployed() error {
 		Status:  "Deployed",
 	}
 
-	return p.writeKymaMetadata(meta)
+	return retryOperation(func() error {
+		return p.writeKymaMetadata(meta)
+	})
 }
 
 func (p *Provider) writeKymaMetadata(data *KymaMetadata) error {
 	cmData := metadataToCM(data)
 
-	//TODO retries
 	kymaMetadataCM, err := p.kubeClient.CoreV1().ConfigMaps(metadataNamespace).Get(context.TODO(), metadataName, metav1.GetOptions{})
```

Comment:
I think you forgot about the retries here?

Reply:
i implemented retries on usages of this method, so the whole method is retried.